scripts/get

```diff
@@ -0,0 +1,145 @@
+#!/bin/sh
```

Comment:
may want to use bash, some keywords (ex: `local`) in this script are not defined in `sh`